jax/_src/scipy/signal.py

```diff
@@ -637,3 +637,31 @@ def istft(Zxx: Array, fs: ArrayLike = 1.0, window: str = 'hann',
 
   time = jnp.arange(x.shape[0], dtype=np.finfo(x.dtype).dtype) / fs
   return time, x
+
+@_wraps(osp_signal.hilbert)
+def hilbert(x, N=None, axis=-1):
+  x = jnp.asarray(x)
```

Comment:
Please call `check_arraylike` before converting to `jnp.array`

Reply:
done